### PR TITLE
Display move target in Doubles and Triples battles

### PR DESCRIPTION
**Pull request by urkerab**
Commit 95c7c4a removed the limited targeting information that had been displayed previously.

### CODE REVIEW

File: js/client-battle-tooltips.js [hunk 1]
```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
 
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
+				if (this.battle.gameType === 'doubles') {
+					if (move.target === 'allAdjacent') {
+							text += '<p class="movetag">&#x2713; Hits both foes and ally.</p>';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: Weird indentation (turns out, this is what Travis was complaining about)

Comment (Zarel) [inline js/client-battle-tooltips.js]: Also would not use checkmark for this. Move targets aren't allies.

Consider `fa-dot-circle-o` or `fa-bullseye`?

File: js/client-battle-tooltips.js [hunk 2]
```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
...
+					} else if (move.target === 'allAdjacentFoes') {
+							text += '<p class="movetag">&#x2713; Hits both foes.</p>';
+					}
+				} else if (this.battle.gameType === 'triples') {
+					if (move.target === 'allAdjacent') {
+						text += '<p class="movetag">&#x2713; Hits adjacent foes and allies.</p>';
+					} else if (move.target === 'allAdjacentFoes') {
+						text += '<p class="movetag">&#x2713; Hits adjacent foes.</p>';
+					} else if (move.target === 'any') {
+						text += '<p class="movetag">&#x2713; Can target distant Pokemon in a Triples battle.</p>';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: `&eacute;`

File: js/client-battle-tooltips.js (hunk 1 above)
Comment (urkerab) [inline js/client-battle-tooltips.js]: I've used `&bull;` for now.

Comment (Zarel) [inline js/client-battle-tooltips.js]: Why `&bull;`?

File: js/client-battle-tooltips.js [hunk 3]
```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
...
+						text += '<p class="movetag">&bull; Hits adjacent foes and allies.</p>';
+					} else if (move.target === 'allAdjacentFoes') {
+						text += '<p class="movetag">&bull; Hits adjacent foes.</p>';
+					} else if (move.target === 'any') {
+						text += '<p class="movetag">&bull; Can target distant Pok&eacute;mon in a Triples battle.</p>';
+					}
+				}
 				if ('defrost' in move.flags) {
-					text += '<p>The user thaws out if it is frozen.</p>';
+					text += '<p class="movetag">&#x2713; Thaws the user out if it is frozen.</p>';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: We only use checkmarks for certain move flags, not all of them. I intentionally don't use them for defrost/protect/authentic.

File: js/client-battle-tooltips.js (hunk 1 above)
Comment (Zarel) [inline js/client-battle-tooltips.js]: Wouldn't something that looks like a bullseye like `fa-dot-circle-o` or `fa-bullseye` look better?

File: js/client-battle-tooltips.js [hunk 4]
```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
 
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
+				if (this.battle.gameType === 'doubles') {
+					if (move.target === 'allAdjacent') {
+						text += '<p class="movetag">&bull; Hits both foes and ally.</p>';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: Wouldn't something that looks like a bullseye like `fa-dot-circle-o` or `fa-bullseye` look better?

File: js/client-battle-tooltips.js [hunk 5]
```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
...
+					} else if (move.target === 'allAdjacentFoes') {
+						text += '<p class="movetag">&bull; Hits both foes.</p>';
+					}
+				} else if (this.battle.gameType === 'triples') {
+					if (move.target === 'allAdjacent') {
+						text += '<p class="movetag">&bull; Hits adjacent foes and allies.</p>';
+					} else if (move.target === 'allAdjacentFoes') {
+						text += '<p class="movetag">&bull; Hits adjacent foes.</p>';
+					} else if (move.target === 'any') {
+						text += '<p class="movetag">&bull; Can target distant Pok&eacute;mon in a Triples battle.</p>';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: I think "in Triples" might be better, like Marty said (?)

File: js/client-battle-tooltips.js (hunk 4 above)
Comment (urkerab) [inline js/client-battle-tooltips.js]: I'll look into fa tomorrow; for now latest push just uses &#x25ce;

File: js/client-battle-tooltips.js [hunk 6]
```diff
@@ -213,7 +213,7 @@ var BattleTooltips = (function () {
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
 				if ('defrost' in move.flags) {
-					text += '<p>The user thaws out if it is frozen.</p>';
+					text += '<p class="movetag">The user thaws out if it is frozen.</p>';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: Out of curiosity, what does the `"movetag"` class do?

Comment (urkerab) [inline js/client-battle-tooltips.js]: I have no idea, I was just making it consistent with the other lines. It's probably there for Stylish users or something.

Comment (Zarel) [inline js/client-battle-tooltips.js]: Actually, because it's copy/pasted from PSdex. We can leave it in for Stylish users.

**Pull request merged**